simplemodels/models.py

```diff
@@ -164,6 +165,36 @@ def _prepare_fields(self, kwargs):
         return kwargs
 
     @classmethod
+    def _get_path(cls, root, path, name):
+        """
+        Helper method to read value from nested data structure by given path.
+
+        :param root: root object with data
+        :param path: beginning of the path
+        :param name: last element name
+        :return: extracted value or None
+        """
+
+        if isinstance(path, six.string_types):
+            path = path.split('/')
+
+        path = list(filter(None, path)) if path else []
+        path = path + [name]
+        cur = root
+        for seg in path:
+            try:
+                cur = cur[seg]
+            except (KeyError, IndexError):
+                return None
+            except TypeError:
+                try:
```

Comment:
Looks good.
If you borrowed [these lines](https://github.com/mahmoud/boltons/blob/4f506cff3cd82ceb07e82b5946190eeada0d75ba/boltons/iterutils.py#L933) as well it would be simplier to get the point =)